File: Chirstmas Tree/tree.c
#include<stdio.h>
int main()
{
    int i,j,k,l=16;
    printf("\n\n\n\n\n\n                          Happy Christmas\n\n\n");
    for(i=0;i<=l;i++)
    {
        for(j=33;j>=i;j--) printf(" ");
        for(k=0;k<=i;k++) printf("* ");
        printf("\n");
    }
    for(i=0;i<(l/3)+1;i++) printf("                                |   |\n");
    printf("                                `````\n\n\n\n");
}

File: Chirstmas Tree/Tree.java
class Tree
{
    public static void main(String[] args) 
    {
        int i,j,l=14;
        System.out.println("\n\n\n\n                    Happy Christmas\n");
        for(i=0;i<=l;i++)
        {
            for(j=27;j>=i;j--) System.out.print(" ");
            for(int k=0;k<i;k++) System.out.print("* ");
            System.out.println();
        }
        for(i=0;i<6;i++)
        System.out.println("                         |   |");
        System.out.println("\n\n\n");
    }
}